test/sagemaker_tests/mxnet/inference/__init__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2018-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import sagemaker
+
+
+
+def invoke_mxnet_helper_function(ecr_image, sagemaker_regions, helper_function, *helper_function_args):
```

Comment:
is there a reason we didn't define one for mxnet training? Also, is there any way we can make a shared utility instead of defining this on a framework level basis?